src/ontology/mondo-ingest.Makefile

```diff
@@ -366,6 +366,6 @@ slurp/%.tsv: $(COMPONENTSDIR)/%.owl $(TMPDIR)/mondo.sssom.tsv $(REPORTDIR)/%_ter
 	--outpath $@
 
 slurp-%:
-	$(MAKE) slurp/$*.tsv
+	$(MAKE) slurp/$*.tsv -B
```

Comment:
@matentzn Adding `-B` should fix it. You went over this w/ me very recently.

Honestly, this should have been pretty obvious to us. Additionally, this is likely an issue for a good many of our `make` goals; that they are not pre-designed to / nor have we (or at least, not I) made it a practice to force update prereqs every time we run them.

I think we will want to put some thought into how we do this. Maybe it's as simple as just remembering to pass `-B` whenever we want to update prereqs. Or maybe there's some other means of foolproofing, or designing variations of goals which either force update prereqs or run normally.

Note that this fix, so to speak, if we want to keep it this way, only applies when running `make slurp-%`. Running `make slurp/%.tsv` will not force update prereqs.